### PR TITLE
Upgrade the docgen client to Github action

### PR DESCRIPTION
**Pull request by karthik-jeevith**
Removed the appveyor.yml and build.ps1 files.
Added Github-cicd.yml file and updated the SDK version

### CODE REVIEW

**Review by avinashbr123 — APPROVED** [review]

File: .github/workflows/github-cd.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: DocumentGeneration Client NuGet CI
+
+env:
+  PROJPATH: './src/Kmd.Logic.DocumentGeneration.Client'
+  DOTNET_VERSION: '3.1.301'
```
Comment (smahapatra86) [inline .github/workflows/github-cd.yml]: Can we not use the latest .net version, 5.0.x?

File: .github/workflows/github-cd.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+on:
+  push:
+    branches: [ master ]
```
Comment (smahapatra86) [inline .github/workflows/github-cd.yml]: `dev` should also be included.

File: .github/workflows/github-cd.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    paths: 
+    - 'src/**'
+  pull_request:
+    branches: [ master ]
```
Comment (smahapatra86) [inline .github/workflows/github-cd.yml]: Missing `dev` branch

File: .github/workflows/github-cd.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        format: YYYYMMDD-HHmm
+
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: ${{ env.DOTNET_VERSION }}
+    - name: Install dependencies
+      run: dotnet restore '${{ env.PROJPATH }}/Kmd.Logic.DocumentGeneration.Client.csproj'
+
+    - name: Build
```
Comment (smahapatra86) [inline .github/workflows/github-cd.yml]: What about build when it is other branches than `master`? Refer to SMS, need to pre-release steps for both `Build` and `Pack` tasks

**Review by smahapatra86 — COMMENTED** [review]
Can you add some description to the PR?

**Review by smahapatra86 — COMMENTED** [review]
Can we try to bump up the docgen client to use .net core sdk 5.0 in this PR?

File: .github/workflows/github-cd.yml (hunk 1 above)
Comment (karthik-jeevith) [reply .github/workflows/github-cd.yml]: Resolved

File: .github/workflows/github-cd.yml (hunk 2 above)
Comment (karthik-jeevith) [reply .github/workflows/github-cd.yml]: Resolved

File: .github/workflows/github-cd.yml (hunk 3 above)
Comment (karthik-jeevith) [reply .github/workflows/github-cd.yml]: Resolved

File: .github/workflows/github-cd.yml (hunk 4 above)
Comment (karthik-jeevith) [reply .github/workflows/github-cd.yml]: Resolved

**Comment by karthik-jeevith** [comment]
> Can you add some description to the PR?

Resolved

**Review by nikhilcr939 — APPROVED** [review]

File: Directory.Build.props [hunk 5]
```diff
@@ -9,7 +9,7 @@
 
   <PropertyGroup>
     <Description>A client library for KMD Logic Document Generation</Description>
-	  <VersionPrefix Condition=" '$(VersionPrefix)' == '' ">2.0.0</VersionPrefix>
+	  <VersionPrefix Condition=" '$(VersionPrefix)' == '' ">3.1.0</VersionPrefix>
```
Comment (smahapatra86) [inline Directory.Build.props]: version prefix seems wrong. Use `2.0.1`

**Review by smahapatra86 — APPROVED** [review]

File: Directory.Build.props (hunk 5 above)
Comment (karthik-jeevith) [reply Directory.Build.props]: Resolved

**Review by avin101 — APPROVED** [review]

**Pull request merged**